data/RTTR/CAMPAIGNS/roman/campaign.ini

```diff
@@ -0,0 +1,10 @@
+[description]
+version=1
+author=Bluebyte
+name=Roman
+short_description=Roman campaign
+long_description=We are in a large dessert with low amount of water and few space for farming. We need to hunt for wild and use our ore mines to produce gold and exchange it for goods we can not produce.
+max_human_players=1
+
+[maps]
+maps=MISS200.WLD,MISS201.WLD,MISS202.WLD,MISS203.WLD,MISS204.WLD,MISS205.WLD,MISS206.WLD,MISS207.WLD,MISS208.WLD,MISS209.WLD,
```

Comment:
just add a `map_dir` and `lua_dir` here in the description pointing to the right directories, then no hardcoding necessary